perfkitbenchmarker/providers/yandexcloud/yc_machine_types.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FLAVORS = [
```

Comment:
Mentioned in another comment but you might want to try this:

```
FLAVORS =  { 'Micro' : MachineConfig(1, 1), 'Small' : MachineConfig(2, 1), ... }
```

So that you can do `memory = FLAVORS['Small'].memory`